### ISSUE TITLE
 Document AssistiveMML extension

### ISSUE THREAD

**Issue by pkra** [mathjax/MathJax-docs#111]
* document the extension
* document expected behavior / limitations, cf https://github.com/mathjax/MathJax/issues/938

Live sample: https://rawgit.com/pkra/ac7d48279728d3ab5881/raw/bb676922f161233224a1aa73b3ba9caf53539225/mathjax-assistivemml.html

**Comment by pkra** [comment]
Draft.


* NVDA:
  *  IE9, Win7
    * works 
    * except that I'm confused. With MathPlayer4 installed, IE9  crashed reliably. After deinstalling MathPlayer, things worked on IE. I thought NVDA needed MathPlayer?
  * IE10, WIn8
    * It looks like aria-hidden fails to be set on the visual output. So NVDA reads both.
    * I ran into similar problems at first (MathPlayer crashing). For some reason, the crashes stopped. I have no idea why though. But once it stopped crashing, the MathML was read correctly.
  * FF 39 (Win7, Win8) works fine
  * Chrome. (Win7, Win8). NVDA skips the math; but the extension works as expected.
* VoiceOver
  *  Yosemite Safari seems to work. 
    * I'm not quite sure what to expect; it reads content several times (but only MathML content as math); that's probably just my inability to navigate. I'm guessing things are alright on our end?
    * will test iOS8 when I get a chance.
*  Jaws 16, latest rev, IE 11, latest patches, win 7 pro, results in them being read as well as Jaws can. Nothing on your end to fix there.

* VoiceOver iOS8
  * the "slide two fingers from top to read screen" method works ok. It reads out the MathML but the highlighting is trying to highlight the hidden MathML.
  * Manual navigation (tapping on an equation, double tapping to go fullscreen) did not work. I'm guessing I couldn't hit that 1px box of the hidden MathML. Tricking it by doing a selection for copying fails badly, reading just the characters from both MathML and MathJax output; this seems expected behavior given how copy&paste strips out markup in general.
  * Overall, the expected results, I think.
  * A critical piece is not mentioned here, which is swiping to it. so take one finger and swipe rapidly from left/right or right/left. You can think of this as tab and shift+tab.

**Comment by pkra** [comment]
Should we document the cookie / hidden menu to trigger it?

**Comment by pkra** [comment]
I've updated the OP. 

Disabling the MathPlayer plugin prevents the crashes but does not fix the disappearance of the aria-hidden attribute (and thus duplicated reading). I've been able to re-attach the attribute on IE11 but NVDA still reads out the visual rendering (after refreshing its buffer).

**Comment by pkra** [comment]
Updated information on JAWS, VoiceOver after more user feedback.

**Comment by pkra** [comment]
Big thanks to @becka11y!

**Comment by dpvc** [comment]
> Should we document the cookie / hidden menu?

Since we are planning to have the extension active in all the combined configurations, it may not be necessary.  It would be useful in the cases where the author has not used a combined configuration file, however, so perhaps it doesn't hurt to mention it.  (Of course, there are a number of other hidden menus that are not documented, so there is precedent for not including it in the docs.)

**Comment by pkra** [comment]
Let's take a more systematic approach.

| Screenreader  | Browser |   OS  | Usable? | Bugs |
| ------------- | ------- | ----- | ------- | ---- |
| ChromeVox    | Chrome | any | +1 | tries to outline hidden MathML |
| NVDA | ANY | WinXP | -1 | cannot install MathPlayer |
| NVDA | Firefox | Win7 | +1 | no bugs |
| NVDA | Firefox | Win8.1 | +1 ||
| NVDA | Firefox | Win10 | +1 |  |
| NVDA | Chrome | WinXP | 0 | |
| NVDA | Chrome | Win7 | 0 | |
| NVDA | Chrome | Win8.1 | 0 |  |
| NVDA | Chrome | Win10 | 0 |  |
| NVDA | MS Edge | Win10 | -1 | [not supported by NVDA] |
| NVDA | IE11 | Win8.1 | +1 | |
| NVDA | IE10 | Win7 | +1 |  |
| VoiceOver | Safari | OSX | +1 | |
| VoiceOver | Chrome | OSX | +1 | |
| VoiceOver | Firefox | OSX | -1 |  |
| JAWS | Firefox |  WinXP | +1 ||
| JAWS | Firefox |  Win7 | +1 | |
| JAWS | Firefox |  Win10 | +1 | |
| JAWS | Chrome |  WinXP | +1 | |
| JAWS | Chrome |  Win7 | +1 | |
| JAWS | Chrome |  Win10 | +1 | |
| JAWS | MS Edge |  Win10 | -1 | [not supported by JAWS]|
| JAWS | IE11 |  Win8.1 | +1 | |
| JAWS | IE10 |  Win8.1 | +1 | no bugs |
| JAWS | IE9 |  Win7 | +1 | no bugs |
| JAWS | FF |  Win8.1 | +1 |  |
| JAWS | Chrome |  Win8.1 | +1 |  |
| JAWS 13 | Chrome |  Win7 | -1 |  [unclear if supported] |
| JAWS 13 | Firefox |  Win7 | -1 |  [unclear if supported] |
| Orca | Firefox | Ubuntu 15.10 | +1 | no bugs |
| Orca | Web | Ubuntu 15.10 | -1 |   [unclear if supported] |
| Orca | Chrome(ium) | Ubuntu 15.10 | -1 | [unclear if supported] |

**Comment by pkra** [comment]
Since edits to my previous posting won't show.

* Figured out NVDA problems (thanks to James -- see notes). All NVDA supported combinations work with the modified AssitiveMML extension (using role=presentation).
* Updated JAWS information
* Updated Orca information.
* some info on MathPlayer.

Overall, it seems that we support everything that we can support. Yay!

I'll make a pull request for the AssistiveMML extension update.
